### DISCUSSION TITLE
Read WaterTemp Via Aqua Computer Highflow Next

### DISCUSSION THREAD

**Discussion by Wayne992** [Rem0o/FanControl.Releases#1152] [General]
Hello,
is there any way i can add the watertemp what my highflow next is reading?
i saw an post that some one did this with the D5 next pump. 
German Theard : https://forum.aquacomputer.de/berwachung-und-steuerung/p1451695-info-steuerung-von-l-ftern-mit-fancontrol-github-jetzt-mit-d5next-support/?highlight=Fancontrol#post1451695
sorry for my bad english =D

**Comment by Fr0stX76** [comment]
Hi there,

Yes  you definitely can by using a plugin.

See the plugin wiki: https://github.com/Rem0o/FanControl.Releases/wiki/Plugins

See the highflownext: plugin: https://github.com/medevil84/FanControl.AquacomputerHighFlowNext

Personally, I prefer using the hwinfo64 plugin to get the temp reading from my highflownext to fancontrol, but either will work.

Hwinfo plugin: https://github.com/Rem0o/FanControl.HWInfo

See the result:

![Capture](https://user-images.githubusercontent.com/70845235/194140167-762dce9d-4eee-4c40-ba1f-02ba14b75be9.jpg)

**Reply by Rem0o** [reply]
@DfrostX Holy config!
